docs/static/v2.0.1-10-geff6f1d/cloud-config.json

```diff
@@ -0,0 +1,415 @@
+{
+ "$schema": "https://kairos.io/v2.0.1-10-geff6f1d/cloud-config.json",
```

Comment:
hmm not sure what happened here, still getting the sha plus why is it getting 2.0.1 :monocle_face: